samples/minimaloboe/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background" />
+    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+</adaptive-icon>
```

Comment:
Add newlines to end of files

Reply:
More default files provided by Android Studio. Should be fixed in Android Studio.